src/standalone/Vegetation/pmodel.jl

```diff
@@ -119,21 +129,20 @@ function PModelConstants{FT}(;
     Hd_Jmax = FT(200000),
     aS_Jmax = FT(659.70),
     bS_Jmax = FT(0.75),
-    Mc = FT(0.0120107),
     oi = FT(0.2095),
     aRd = FT(0.1012),
     bRd = FT(-0.0005),
     fC3 = FT(0.015),
 ) where {FT <: AbstractFloat}
-
+    earth_param_set = LP.LandParameters(FT)
     return PModelConstants{FT}(
-        R,
+        LP.gas_constant(earth_param_set),
         Kc25,
         Ko25,
-        To,
+        FT(298.15),
```

Comment:
We should be able to get this from ClimaParams using:
```suggestion
        LP.get_default_parameter(:kelvin_25C)
```